test/extended/operators/operators.go

```diff
@@ -101,68 +101,65 @@ var _ = g.Describe("[sig-arch][Early] Managed cluster should", func() {
 				return false, nil
 			}
 
-			var unavailable []objx.Map
-			var unavailableNames []string
+			var working []string
 			for _, co := range items {
-				if condition(co, "Available").Get("status").String() != "True" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
-				}
-				if condition(co, "Progressing").Get("status").String() != "False" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
-				}
-				if condition(co, "Failing").Get("status").String() != "False" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
+				conditions, missingTypes := surprisingConditions(co)
```

Comment:
```suggestion
				badConditions, missingConditions := surprisingConditions(co)
```

Reply:
Fixed in fe2a717 -> 5819963.

Reply:
i see the change elsewhere but not on this line?

Reply:
Fixed with 5bbe680d26 -> d7a7ff790f.